### ISSUE TITLE
Why can't users with the role of Drive User create a group?

### ISSUE THREAD

**Issue by Dieule26** [frappe/drive#222]
Only Drive Admin role can create groups. I think users should also have the ability to create groups.

**Comment by Dieule26** [comment]
I also agree with you that allowing everyone to create groups can lead to the creation of too many unnecessary groups. However, I think any Drive user should have the right to create groups. Each user will have different needs for creating groups to quickly share documents. If only administrators have the right to create groups, it will be very inconvenient.

**Comment by safwansamsudeen** [comment]
User groups have been removed, largely made redundant by Teams.
